addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxPlayerHandler.java

```diff
@@ -254,12 +288,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void playerAdded(SqueezeBoxPlayer player) {
         if (isMe(player.getMacAddress())) {
```

Comment:
Agreed.  I think it also may require some changes to EventListener and ServerHandler.  Let me look at this.